### PR TITLE
OC-617 ~ Fixes errors in smart metering monitoring WSDL.

### PR DESCRIPTION
**Pull request by kevinsmeets**
The smart metering monitoring WSDL contains 2 errors:

- The part 'GetPowerQualityProfileRequest' has an invalid value 'GetPowerQualityProfileRequest' defined for its element. Element declarations must refer to valid values defined in a schema.
- The part 'GetPowerQualityProfileResponse' has an invalid value 'GetPowerQualityProfileResponse' defined for its element. Element declarations must refer to valid values defined in a schema.

Issue reported: https://github.com/OSGP/Documentation/issues/230

### CODE REVIEW

**Comment by coendamen** [comment]
for my info, what is the error here that needs to be fixed ?

**Review by harrymiddelburgetall — APPROVED** [review]
Voor dit soort class imports zou je de mogelijkheid van een alias wensen voor de leesbaarheid

File: osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/GetPowerQualityProfileRequestMappingTest.java [hunk 1]
```diff
@@ -33,7 +32,7 @@
 
     @Test
     public void convertGetPowerQualityProfileRequestData() {
```
Comment (coendamen) [inline osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/GetPowerQualityProfileRequestMappingTest.java:34]: also change code to remove "Data" ?

File: osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/GetPowerQualityProfileRequestMappingTest.java [hunk 2]
```diff
@@ -33,7 +32,7 @@
...
-        final GetPowerQualityProfileRequestData source = this.makeRequest();
+        final org.opensmartgridplatform.adapter.ws.schema.smartmetering.monitoring.GetPowerQualityProfileRequest source = this.makeRequest();
         final Object result = this.mapper.map(source, GetPowerQualityProfileRequest.class);
 
         assertThat(result).as("mapping GetPowerQualityProfileRequestData should not return null").isNotNull();
```
Comment (coendamen) [inline osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/GetPowerQualityProfileRequestMappingTest.java:38]: also change code to remove "Data" ?

**Review by coendamen — APPROVED** [review]
one small remark. change looks good

File: osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/GetPowerQualityProfileRequestMappingTest.java (hunk 2 above)
Comment (jboon) [reply osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/GetPowerQualityProfileRequestMappingTest.java:38]: the text in the assert right?

**Pull request merged**